governance/votes/Executive vote - October 04, 2024.md

```diff
@@ -0,0 +1,158 @@
+
+### LITE-PSM-USDC-A Migration Phase 3
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183/2), [Poll 1143](https://vote.makerdao.com/polling/QmRjrFYG)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183)
+
+If this executive proposal passes, then the [third and final set of actions](https://forum.sky.money/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183) in the [migration process from USDC-PSM-A to LITE-PSM-USDC-A](https://forum.sky.money/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512) will be executed. This set of actions comprises the following:
+
+- Update PSM-USDC-A fees.
+- Migrate all remaining USDC reserves from PSM-USDC-A to LITE-PSM-USDC-A.
+- Update the PSM-USDC-A [Debt Ceiling Instant Access Module (DC-IAM)](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).
+- Increase the [LITE-PSM-USDC-A Buffer (`buf`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.6_Buffer_Definition-cf8e3d20-fffa-4b6a-9bf2-169d493c3638|57ea599773d45150b2632df9).
+- Update the LITE-PSM-USDC-A [DC-IAM](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).
+
+These actions are described below.
+
+#### PSM-USDC-A Fee Update
+
+- Decrease [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) by 0.01 percentage points from 0.01% to **0%**.
+- Decrease [Fee Out (`tout`)](https://manual.makerdao.com/module-index/module-psm#fee-out-tout) by 0.01 percentage points from 0.01% to **0%**.
+
+#### Remaining USDC Migration
+
+The remaining USDC in [PSM-USDC-A](https://makerburn.com/#/collateral/PSM-USDC-A) will be migrated to [LITE-PSM-USDC-A](https://makerburn.com/#/collateral/LITE-PSM-USDC-A) through a script executed in the spell.
+
+#### PSM-USDC-A DC-IAM Update
+
+- Decrease the [Maximum Debt Ceiling (`line`)](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4.1_Maximum_Debt_Ceiling_(line)-6f1a913d-9436-4b70-816b-e317672737d6|57eaf45219bea3b430c268bb) by 2,5 billion DAI from 2,5 billion DAI to **0 DAI**.
+- Disable the PSM-USDC-A DC-IAM.
+
+#### LITE-PSM-USDC-A Buffer Increase
+
+- Increase the [LITE-PSM-USDC-A Buffer (`buf`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.6_Buffer_Definition-cf8e3d20-fffa-4b6a-9bf2-169d493c3638|57ea599773d45150b2632df9) by 200 million DAI from 200 million DAI to **400 million DAI**.
+
+#### LITE-PSM-USDC-A DC-IAM Update
+
+- Increase the [Maximum Debt Ceiling (`line`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.3_Maximum_Debt_Ceiling_Definition-e4a0d8f1-4a01-46a7-b693-a94e7f4afc1a|57ea599773d45150b2632df9) by 2,5 billion DAI from 7,5 billion DAI to **10 billion DAI**.
```

Comment:
Let's be consistent with decimal notation. For the Parameter Change section above you've used `.` but here you've used `,` We suggest changing all to `.`

```suggestion
- Decrease the [Maximum Debt Ceiling (`line`)](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4.1_Maximum_Debt_Ceiling_(line)-6f1a913d-9436-4b70-816b-e317672737d6|57eaf45219bea3b430c268bb) by 2.5 billion DAI from 2.5 billion DAI to **0 DAI**.
- Disable the PSM-USDC-A DC-IAM.

#### LITE-PSM-USDC-A Buffer Increase

- Increase the [LITE-PSM-USDC-A Buffer (`buf`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.6_Buffer_Definition-cf8e3d20-fffa-4b6a-9bf2-169d493c3638|57ea599773d45150b2632df9) by 200 million DAI from 200 million DAI to **400 million DAI**.

#### LITE-PSM-USDC-A DC-IAM Update

- Increase the [Maximum Debt Ceiling (`line`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.3_Maximum_Debt_Ceiling_Definition-e4a0d8f1-4a01-46a7-b693-a94e7f4afc1a|57ea599773d45150b2632df9) by 2.5 billion DAI from 7.5 billion DAI to **10 billion DAI**.
```